csi.proto

```diff
@@ -456,6 +456,17 @@ message ListVolumesResponse {
 message GetCapacityRequest {
   // The API version assumed by the CO. This is a REQUIRED field.
   Version version = 1;
+
+  // If specified, the Plugin SHOULD only report the capacity of the
```

Comment:
If a plugin can't determine capacity, then it should say so via controller capabilities, right?